SEP-0006.md

```diff
@@ -0,0 +1,38 @@
+* author: Steven Christe
+* contact email:
+* date-creation: 2015-09-23
+* date-last-revision: 2015-09-23
+* type: process
+* status: review
+
+# Abstract
+The purpose of this document is contains the membership information of
+the SunPy board, including names as well as membership start, and expiration
+date and board roles. Additionally this document will name the Lead developer.
+
+# Detailed Description
+There is no official place to list the current membership of the SunPy board
+or the lead developer. This document shall be referenced by the board founding document (SEP-00001).
+This information shall be posted on the official sunpy website for added transparency.
+
+# Board Membership
+The SunPy board is composed of the following members (in alphabetical order)
+
+| Name                          | Role              | Start       |  Expiration |
+|-------------------------------|-------------------|-------------|-------------|
+| Steven Christe                | Chair             | 17 Mar 2014 | 31 Dec 2016 |
+| Russell Hewett                |                   | 17 Mar 2014 | 31 Dec 2015 |
+| Andrew Inglis                 | Secretary         | 17 Mar 2014 | 31 Dec 2015 |
+| Jack Ireland                  |                   | 17 Mar 2014 | 31 Dec 2016 |
+| Stuart Mumford                |                   | 17 Mar 2014 | 31 Dec 2015 |
```

Comment:
That is not a board role I think. The board appoints a director/lead developer but that person does not need to be on the board.